src/utils/linking.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import { Linking } from 'react-native';
+import t from 'translations/translate';
+
+// Components,
+import Toast from 'components/Toast';
+
+// Services
+import smartWalletService from 'services/smartWallet';
+
+export function viewOnBlockchain(hash: ?string) {
```

Comment:
I would move this to `common` or `transactions` utils as `linking` doesn't say much/reflect the contents and it would most likely will contain this method only

Reply:
Linking refers to the fast that it uses `Linking` utility from React Native. I'll try to figure out something more descriptive. I didn't want to move this to `common` as `common` is IMO too messy atm with all the various staff that's there.